swan/values.yaml

```diff
@@ -118,7 +118,8 @@ jupyterhub:
     # placeholder for hub secret token
     secretToken:
   hub:
-    fsGid: 0
+    podSecurityContext:
+      fsGroup: 0
```

Comment:
Did you end up trying to deploy without this? I don't remember what was the conclusion of our discussion (if to leave it or not). But let's document it here in this PR, at least.

Reply:
No I did not try without in the end. I added this task to do it in the future: https://its.cern.ch/jira/browse/SWAN-92